test/verify/check-system-info

```diff
@@ -380,5 +380,27 @@ class TestSystemInfo(MachineCase):
         b.enter_page("/system")
 
 
+    def testPcpAvailable(self):
+        m = self.machine
+        b = self.browser
+
+        self.login_and_go("/system")
+
+        # Don't install/uninstall pcp in this test, so that we avoid hitting
+        # the network. Instead, check that the link to install pcp is there
+        # when it is not installed.
+        installed = m.execute("systemctl show -p LoadState pmlogger") == "LoadState=loaded"
```

Comment:
TBH I'd prefer having a static list of OSes here. I. e. at least "debian-stable", I'm not sure about the Atomics. Every other OS ought to already have it installed.

So this should check that `Enable...` is *not* shown on any OS by default, as all our images which *can* have pcp now do (i. e. all but debian-stable). Then everywhere but debian-stable you can remove the pcp package, reload the page, and check that you now get the `Enable...` offered. This doesn't need networking.